tests/onnxruntime/test_modeling.py

```diff
@@ -1386,8 +1386,10 @@ def test_pipeline_on_gpu(self, model_arch):
         # check model device
```

Comment:
should be added 1 line above: 
We can also add `generation_args` to have a `min_lenght` for the test. to pass?